format.html

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Format
+description: Arrow Format
+---
+
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+
+<div class="row">
+  <div class="col-md-6">
+    <h2>Performance Advantage of Columnar In-Memory</h2>
```

Comment:
Why does some text have to be written out as HTML, rather than Markdown?

Reply:
It probably doesn't (though this file would have to be renamed to .md to be rendered). There are some things the markdown renderer can't do right, but this would probably be fine. This was just copied from the original homepage and was already html.